README.md

```diff
@@ -62,9 +60,12 @@ You can setup storages either at zenoh router startup via a configuration file,
               // URL to the InfluxDB service
               url: "http://localhost:8086",
               private: {
-                // If needed: InfluxDB credentials, preferably admin for databases creation and drop
+                // If needed: InfluxDB v1 credentials, preferably admin for databases creation and drop
                 //username: "admin",
                 //password: "password"
+                // Required: InfluxDB v2 token, preferably admin for databases creation and drop
+                //NOTE: For influxdb v2.x, the influxdb rust library currently doesn't provide for creating and dropping the database but it is still required to keep a token value here
```

Comment:
Weird wording... *doesn't provide for*? Did you mean *doesn't provide SUPPORT for*?